flight/logger.py

```diff
@@ -0,0 +1,52 @@
+"""Logging configuration and functions"""
+import logging
+from typing import TextIO
+from datetime import datetime
+from multiprocessing import Queue
+from logging import Formatter, FileHandler, StreamHandler
+from logging.handlers import QueueHandler, QueueListener
+
+LOG_FILE: str = f"logs/{datetime.now()}.log"
+LOG_FORMAT: str = "%(levelname)s | %(asctime)s @  %(processName)s:%(funcName)s > %(message)s"
+LOG_LEVEL = logging.DEBUG
+
+
+def init_logger(queue: Queue[str]) -> QueueListener:
+    """
+    Creates a QueueListener that will process all log messages throughout the application
+
+    Parameters
+    ----------
+    queue: Queue
+        Queue object that holds logging processes
+
+    Returns
+    -------
+    queuelistener: QueueListener
+        Object to process log messages
+    """
+    console_formatter: Formatter = logging.Formatter(LOG_FORMAT)
+    file_formatter: Formatter = logging.Formatter(LOG_FORMAT)
+
+    file: FileHandler = logging.FileHandler(LOG_FILE, "a")
```

Comment:
This is kinda a nitpick, but change the output file name to be something other than "a"

Reply:
The file name is the LOG_FILE variable it is log/<datetime>.log and a is just the output mode 😺 ☆*: .｡. o(≧▽≦)o .｡.:*☆